packages/grafana-data/src/types/vector.ts

```diff
@@ -10,6 +44,18 @@ export interface Vector<T = any> {
    * Get the results as an array.
    */
   toArray(): T[];
+
```

Comment:
These 4 new functions are are *real* breaking change... this means that anyone who has implemented Vector directly rather than use any of the existing Vector implementations would now be invalid.

We could avoid it, but then typing gets weird and we can not use these functions on `T[] | Vector<T>` 🤷